macro-fullcalendar-api/src/main/java/org/xwiki/fullcalendar/FullCalendarManager.java

```diff
@@ -38,4 +40,16 @@ public interface FullCalendarManager
      * @throws Exception in case of exceptions.
      */
     String iCalToJSON(String iCalStringURL) throws Exception;
+
+    /**
+     * Convert an iCal to a JSON.
```

Comment:
I agree with @lucaa that we should state more clearly that this method is in fact extracting the events between the given start and end dates, so it's not a full representation of the specified iCall as JSON.